tests/integration/storage_test.py

```diff
@@ -0,0 +1,49 @@
+"""Integration tests for google.datalab.storage."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+import random
+import string
+import subprocess
+import unittest
+
+import google.datalab
+from google.datalab import storage
+
+
+class StorageTest(unittest.TestCase):
+
+  def setUp(self):
+    project_id = os.environ.get('PROJECT_ID', '')
+    if not project_id:
+      # If there's no project ID set, we try asking gcloud. We need to send
+      # stderr to /dev/null, otherwise we get some spurious output.
+      gcloud_output = subprocess.check_output([
+          'gcloud', 'config', 'get-value', 'project'], stderr=subprocess.PIPE)
+      project_id = gcloud_output.strip()
+    logging.info('Using project ID: %s', project_id)
+
+    context = google.datalab.Context.default()
+    context.set_project_id(project_id)
```

Comment:
Why don't we just use `google.datalab.Context.default()` as is? It should pick up the project value from gcloud, just like what the lines 21-31 do.